### DISCUSSION TITLE
<title>Github application to join

### DISCUSSION THREAD

**Discussion by Stacey1160** [community/community#45489] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have been trying for the past 2 weeks to enroll for the GitHub service pack but every time I put my school's URL in it keeps telling me to put my school's URL in when I click Continue.  I sent a message before but I don't remember if I got a reply or not.  I'm a student at MyComputerCareer.edu studying Information Technology.

**Comment by jge162** [comment]
If approved you will get an email. If not approved they won’t email you. If it has been two weeks does not appear you were approved. See below for more help. 

https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved
